package-res/resources/web/pentaho/type/_doc/spec/ITypeConfiguration.jsdoc

```diff
@@ -0,0 +1,100 @@
+/*!
+ * Copyright 2010 - 2016 Pentaho Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * The `spec.ITypeConfiguration` interface describes
+ * a list of [configuration rules]{@link pentaho.type.spec.ITypeConfiguration#rules}
+ * used to configure one or more [value types]{@link pentaho.type.Value.Type} of
+ * the Pentaho Client Metadata Model.
+ *
+ * The following example is an hypothetical type configuration,
```

Comment:
Change "an" to "a" in front of hypothetical, also see suggested rewording:

The following example is a hypothetical configuration, where a developer (John) states that only he should see the visualization he is currently developing (my/radial/bar) and only then when working in Pentaho Analyzer: